### PR TITLE
Set up logging to a file

### PR DESCRIPTION
**Pull request by lbarcziova**
`/log-files` structure:
├── repo1
│   ├── 0a0c838_2020-11-10T10:19:06.log
│   └── af0c838_2020-11-10T10:19:06.log
├── repo2
│   ├──af0c838_2020-11-10T10:19:06.log

Logs format:
`[2020-11-09 16:09:04,845 logging.py INFO] Processing repository repo1, commit SHA 0a0c838.`

### CODE REVIEW

File: dist2src/worker/logging.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    :param repo_name: name of the repository
+    :param commit_sha: commit SHA
+    :param logs_dir: dir where the logs are stored
+    :return:
+    """
+    logger = logging.getLogger("dist2src")
+    logger.setLevel(logging.DEBUG)
+    log_dir = logs_dir / repo_name
+    log_dir.mkdir(parents=True, exist_ok=True)
+    filename = f"{commit_sha}_{datetime.now().strftime('%Y-%m-%d_%H-%M-%S')}.log"
```
[the remark this answers is not in the document]
Comment (lbarcziova) [reply dist2src/worker/logging.py]: thanks for the suggestion :+1:  changed

**Comment by lbarcziova** [comment]
recheck

**Review by TomasTomecek — APPROVED** [review]

**Comment by TomasTomecek** [comment]
recheck won't help here, the test failure is going to be solved after #120 is in

hence I'd merge it like this

**Comment by lbarcziova** [comment]
> recheck won't help here, the test failure is going to be solved after #120 is in
hence I'd merge it like this

thanks for the info, merging it then

**Pull request merged**